inference-engine/src/cldnn_engine/cldnn_program.cpp

```diff
@@ -173,9 +173,10 @@ void Program::CleanupBuild() {
     m_networkOutputs.clear();
 }
 
-std::shared_ptr<cldnn::program> Program::BuildProgram(std::vector<std::shared_ptr<ngraph::Node>> ops,
+std::shared_ptr<cldnn::program> Program::BuildProgram(const std::vector<std::shared_ptr<ngraph::Node>>& ops,
                                                       InferenceEngine::InputsDataMap networkInputs,
-                                                      InferenceEngine::OutputsDataMap networkOutputs) {
+                                                      InferenceEngine::OutputsDataMap networkOutputs,
+                                                      bool build) {
```

Comment:
how about make it as opposite to exit first if it is expected only create topology? 

`BuildProgram(..., bool create_only_topology) {
...
  for( op : ops) {
      CreateSingle...
  }

  if (create_only_topology) return {};

  auto program = make_shared<program>(..);

  return program;
}

Reply:
Fixed

Reply:
hmm? is it updated?

Reply:
I just pushed the commit, you can see the update now :)